src/devices/machine/74259.h

```diff
@@ -112,6 +110,9 @@ class addressable_latch_device : public device_t
 	devcb_write_line    m_q_out_cb[8];  // output line callback array
 	devcb_write8        m_parallel_out_cb; // parallel output option
 
+	// miscellaneous configuration
+	bool    m_clear_active;             // active state of clear line
```

Comment:
This member should be `const` - it shouldn't ever get changed after construction since it's an inherent part of the device.